.travis/linters.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 set -exu
 
-go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
-go install github.com/golangci/golangci-lint/cmd/golangci-lint
+GO111MODULE=on
+# installing golangci-lint as recommended on the project page
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin latest
```

Comment:
there were go-get issues so I switched to the recommended way to install golangci-lint

Reply:
wow `curl | sh` /o\

Reply:
Yeah I know, but it's over https, from a known github link pattern, so as long as you trust the developer, that is not worse than `go get`